src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSectionRootRenderer.cs

```diff
@@ -211,7 +211,7 @@ protected virtual void LayoutRenderers()
 				var shellContent = items[i];
 				if (_renderers.TryGetValue(shellContent, out var renderer))
 				{
-					var view = renderer.PlatformView;
+					var view = renderer.ViewController.View;
```

Comment:
Here's the only additional change from the original PR https://github.com/dotnet/maui/pull/13332